infrastructure/prime-app-ephemeral-template.yml

```diff
@@ -531,7 +524,10 @@ objects:
               - createdb -O $(APPUSER) prime-${SVC_NAME}
             env:
               - name: PGHOST
-                value: dev-patroni
+                valueFrom:
+                  configMapKeyRef:
+                    name: database-cm
+                    key: database-host
```

Comment:
Can `database-cm` be named something more intuitive?

Reply:
Also, are these ConfigMaps needed:  `dev-database`, `dev-patroni-config`, `dev-patroni-failover`, `dev-patroni-leader`

Reply:
changed `database-cm` to using `dev-databsae`, same as in non-ephemeral template.

Patroni uses config maps to store its parameters and for leader election. dev-patroni-* config maps are absolutely necessary in our case.